core/templates/pages/splash-page/splash-page.component.spec.ts

```diff
@@ -88,7 +88,7 @@ describe('Splash Page', function() {
     ctrl.onClickBrowseLibraryButton();
     $timeout.flush(150);
 
-    expect(windowRefMock.nativeWindow.location).toBe('/community-library');
+    expect(windowRefMock.nativeWindow.location).toBe('/learn/math');
```

Comment:
Change the test name i.e, should redirect to the default classroom page. Should we use constants.DEFAULT_CLASSROOM_URL_FRAGMENT for testing?